src/Helpers/LiFiData.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+
+pragma solidity ^0.8.17;
+
+/// @title LiFiData
+/// @author Li.Finance (https://li.finance)
+/// @notice A storage for LI.FI-internal config data (addresses, chainIDs, etc.)
+/// @custom:version 1.0.0
+contract LiFiData {
+    address internal constant NON_EVM_ADDRESS =
```

Comment:
isn't it duplication from `LibAsset.NON_EVM_ADDRESS` ?

Reply:
moved to LiFiData as discussed